files/aws_lifecycle_hooks/attach_ebs_volume.py

```diff
@@ -11,14 +9,21 @@
 from exceptions import VolumeInUseError
 
 
+def device_mounted(device: str) -> bool:
+    ebs_mountpoints = tools.get_block_device_mountpoint('ebs[0-9]+')
+    if device in ebs_mountpoints:
```

Comment:
There is no else:-branch, in which case you implicitly return None, and not False.
I'd suggest to do `return (device in ebs_mountpoints)`, which gets rid of the if-statement and does the right thing.